script/predicate_functions.sh

```diff
@@ -23,18 +23,6 @@ function is_mri_192 {
   fi
 }
 
-function is_mri_192_plus {
-  if is_mri; then
-    if ruby -e "exit(RUBY_VERSION.to_f > 1.9)"; then
-      return 0
-    else
-      return 1
-    fi
-  else
-    return 1
-  fi
-}
-
```

Comment:
Not an expert here, but maybe a git submodule could be used to prevent having this kind of external files committed to version control in several places?